### PR TITLE
added ForceEndManipulation to ManipulationHandler

### PR DESCRIPTION
**Pull request by thalbern**
## Overview
ManipulationHandler now has a ForceEndManipulation that will end the currently active manipulation state and release the object
- change includes sample scene and playmode unit test

## Changes
- Fixes: #4584 

## Verification
Added sample scene that shows functionality
mainpulationhandlerplayground

Moving sphere into the wall will call ForceEndManipulation

manipulation_handler_force_release

### CODE REVIEW

**Comment by thalbern** [comment]
just trying if i can add a test as well for this - will update review soon if i manage :D

File: Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs [hunk 1]
```diff
@@ -99,6 +99,76 @@ public IEnumerator Test02_ManipulationHandlerGazeHover()
...
+        /// <summary>
+        /// Test creates an object with manipulationhandler verifies translation with articulated hand
+        /// as well as forcefully ending of the manipulation
+        /// from gaze provider.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator ManipulationHandlerForceRelease()
+        {
+            TestUtilities.InitializeMixedRealityToolkitAndCreateScenes(true);
```
Comment (lukastoenneMS) [inline Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs]: I believe this happens in SetupMrtk already, these two lines should not be needed.

File: Assets/MixedRealityToolkit.Examples/Demos/UX/ManipulationHandler/Scripts/ChangeManipulation.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                manipulatedObjCollider = manipulatedObject.GetComponent<Collider>();
+            }
+        }
+
+        private void Update()
+        {
+            TryStopManipulation();
+        }
+
+        public void TryStopManipulation()
```
Comment (julenka) [inline Assets/MixedRealityToolkit.Examples/Demos/UX/ManipulationHandler/Scripts/ChangeManipulation.cs]: Nit: could you use trigger volumes instead? like stop manipulation in [OnTriggerEnter](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnTriggerEnter.html)?

File: Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs [hunk 3]
```diff
@@ -99,6 +99,73 @@ public IEnumerator Test02_ManipulationHandlerGazeHover()
...
+
+            // grab the cube - move it to the right 
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+            int numSteps = 30;
+            
+            Vector3 handOffset = new Vector3(0, 0, 0.1f);
+            Vector3 initialHandPosition = new Vector3(0, 0, 0.5f);
+            Vector3 rightPosition = new Vector3(1f, 0f, 1f);
+
+            yield return PlayModeTestUtilities.ShowHand(Handedness.Right, inputSimulationService);
```
Comment (julenka) [inline Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs]: hehe these are already being used, yay!

File: Assets/MixedRealityToolkit.Examples/Demos/UX/ManipulationHandler/Scripts/ChangeManipulation.cs (hunk 2 above)
Comment (thalbern) [reply Assets/MixedRealityToolkit.Examples/Demos/UX/ManipulationHandler/Scripts/ChangeManipulation.cs]: looked into this, but I can't change it easily bc we're not receiving any of the collision events when we're manipulating the object - need to look into how to enable those events in another PR first

**Pull request merged**